Compiler/src/proto/operations_pb.ml

```diff
@@ -234,6 +234,54 @@ let default_eval_mutable () : eval_mutable = {
   code = "";
 }
 
+type begin_class_definition_mutable = {
```

Comment:
Just to double check, all the code in the src/proto directory is automatically generated by the `build-pbs.sh` script from the protobuf definitions in Fuzzilli?

Reply:
Yes it is! `esy run-script build-pbs` generates all the files in `src/proto/`, using the protobuf files in the main Fuzzilli